File: my-submissions/m2997.java
class Solution {
    public int minOperations(int[] nums, int k) {
        for (int num : nums) {
            k ^= num;
        }

        int counter = 0;
        while (k > 0) {
            if (k % 2 == 1) {
                counter++;
            }
            
            k = k >> 1;
        }

        return counter;
    }
}

File: my-submissions/m2997.py
class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        xored = k
        for num in nums :
            xored ^= num

        count = 0
        while xored > 0 :
            if xored % 2 == 1 :
                count += 1
            
            xored = xored >> 1
        return count